extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -63,14 +64,13 @@ class QueueProcessor {
         this.sourceHTTPAgent = new http.Agent({ keepAlive: true });
         this.destHTTPAgent = new http.Agent({ keepAlive: true });
 
-        // FIXME support multiple destination sites
-        if (destConfig.bootstrapList.length > 0) {
-            this.destHosts =
-                new RoundRobin(destConfig.bootstrapList[0].servers,
-                               { defaultPort: 80 });
-        } else {
-            this.destHosts = null;
-        }
+        // FIXME support multiple cloud-server destination sites
+        destConfig.bootstrapList.forEach(dest => {
```

Comment:
in the config.joi, `bootstrapList` is not a required field, so using the `forEach` here might cause a problem. Maybe we can guarantee by making `bootstrapList` a required field in the config.joi file? Are there scenarios where `bootstrapList` is not used?

Also using `forEach` here might not be best because that if we have multiple destination servers, we would be looping through all of them and setting a new `RoundRobin` each time

Reply:
I was going off the fact that we were already checking the length of `bootstrapList`. But I will add a conditional here to be sure.

I've included the 'FIXME' comment to refer to the second case, since we don't support multiple desintation servers anyway, I think it's okay for now to assume that the `bootstrapList` object with a `servers` property is the one that we want.